src/node.cc

```diff
@@ -840,6 +849,24 @@ void SetupNextTick(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(ret);
 }
 
+void WeakCallback(
+    const v8::WeakCallbackInfo<v8::Persistent<v8::Promise,
+      node::ResetInDestructorPersistentTraits<v8::Promise> > >& data) {
```

Comment:
We don’t need `ResetInDestructorPersistentTraits` if we use `Persistent` or `node::Persistent` instead of `v8::Persistent`, right?

Also, for a function in `node.cc` the name is a bit too generic imo…